### ISSUE TITLE
Configuration UI unusable

### ISSUE THREAD

**Issue by G1LL1ES** [custom-cards/spotify-card#183]
**Describe the bug**
Dropdowns not functioning due to recent HA release

* [x] Reproducable on a plain lovelace panel without card, style or theme mods
* [x] Using latest version of spotcast and spotify-card [ ]
* [x] Using latest stable version of Home Assistant
* [x] I have Spotify Premium
* [x] I have setup the Spotify integration in Home Assistant

**To Reproduce**
1. Add new Custom: Spotify Card to dashboard
2. Observe configuration UI

**Expected behavior**
Config UI would correctly display dropdowns

**Screenshots**
![SpotifyCardBug](https://user-images.githubusercontent.com/32212151/167982743-5d11f696-8761-491a-b1b1-60ebdd52ce63.png)

**Environment (please complete the following information):**
 - OS: Windows 10
 - Browser: Chrome
 - Version: 101.0.4951.54
 - HA version: Home Assistant Core 2022.5.3
 - spotcast version: 3.6.29

**Comment by seraphx2** [comment]
Yeah. There is definitely a lot broken in the UI. Just ended up configuring it all by hand.

**Comment by woempiej** [comment]
Maybe the card is coming to an end

**Comment by arretx** [comment]
Same problem.  Yaml mode only option it seems.
